### PR TITLE
fix(medusa): Only add ordering select if not already present

### PR DESCRIPTION
**Pull request by pKorsholm**
**What** 
- fix bug where adding `location_id` as a queryparam for list-inventory-item would break list-inventory-item due to ambiguous reference

### CODE REVIEW

**Review by carlos-r-l-rodrigues — APPROVED** [review]
LGTM

File: packages/inventory/src/utils/query.ts [hunk 1]
```diff
@@ -65,7 +65,9 @@ export function getListQuery(
     const toSelect: string[] = []
     const parsed = Object.entries(query.order).reduce((acc, [k, v]) => {
       const key = `inv_item.${k}`
-      toSelect.push(key)
+      if((query.select && !query.select[k] ) || !query.select){
```
Comment (adrien2p) [inline packages/inventory/src/utils/query.ts:68]: todo: there is a lint issue :D

Comment (adrien2p) [reply packages/inventory/src/utils/query.ts:68]: suggestion: 
```ts
if (!query.select?.[k] ) || !query.select) {
 // ...
}
```

**Review by olivermrbl — APPROVED** [review]
LGTM

**Pull request merged**